tests/slice_plotter_presenter_test.py

```diff
@@ -17,34 +17,35 @@ def setUp(self):
         # Setting up view for slice plotter window
         self.slice_plotter = mock.create_autospec(SlicePlotter)
         self.slice_view = mock.create_autospec(SlicePlotterView)
+        self.plotting_module = mock.Mock(spec=["imshow","xlabel","ylabel"])
```

Comment:
This highlights there is no formally defined interface for the module (i am passing in the module because I need 4 functions, and those would be cumbersome to pass is as arguments)
Is this ok? 
Should I create a class containing a subset of the api needed? 
Or would passing in a tuple of parameters suffice?